PWGHF/TableProducer/candidateCreatorXic0Omegac0.cxx

```diff
@@ -680,109 +680,146 @@ struct HfCandidateCreatorXic0Omegac0 {
       KFV0.GetMass(massLam, sigLam);
       if (TMath::Abs(massLam - MassLambda0) > lambdaMassWindow)
         continue;
-      registry.fill(HIST("hKFParticleV0Chi2"), KFV0.GetChi2());
-      if (kfUseV0MassConstraint) {
-        KFV0.SetNonlinearMassConstraint(o2::constants::physics::MassLambda);
-      }
-
+      // err_mass>0 of Lambda
+      if (sigLam <= 0)
+        continue;
+      // chi2>0 && NDF>0 for selecting Lambda
+      if ((KFV0.GetNDF() <= 0 || KFV0.GetChi2() <= 0))
+        continue;
+      kfOmegac0Candidate.chi2GeoV0 = KFV0.GetChi2();
       KFParticle KFV0_m = KFV0;
       KFV0_m.SetNonlinearMassConstraint(o2::constants::physics::MassLambda);
-
-      //-------------------------- V0 info---------------------------
-      // pseudorapidity
-      float pseudorapV0Dau0 = trackV0Dau0.eta();
-      float pseudorapV0Dau1 = trackV0Dau1.eta();
-
-      // info from from KFParticle
-      std::array<float, 3> pVecV0 = {KFV0.GetPx(), KFV0.GetPy(), KFV0.GetPz()}; // pVec stands for vector containing the 3-momentum components
-      std::array<float, 3> vertexV0 = {KFV0.GetX(), KFV0.GetY(), KFV0.GetZ()};
-      std::array<float, 3> pVecV0Dau0 = {kfpPos.GetPx(), kfpPos.GetPy(), kfpPos.GetPz()};
-      std::array<float, 3> pVecV0Dau1 = {kfpNeg.GetPx(), kfpNeg.GetPy(), kfpNeg.GetPz()};
+      if (kfUseV0MassConstraint) {
+        KFParticle KFV0 = KFV0_m;
+      }
 
       //__________________________________________
-      //*>~<* step 2 : reconstruc cascade(Omega) with KF
-      KFParticle kfpV0 = KFV0;
-      const KFParticle* OmegaDaugthers[2] = {&kfpBachKaon, &kfpV0};
+      //*>~<* step 2 : reconstruct cascade(Omega) with KF
+      const KFParticle* OmegaDaugthers[2] = {&kfpBachKaon, &KFV0};
```

Comment:
```suggestion
      const KFParticle* omegaDaugthers[2] = {&kfPBachKaon, &kfV0};
```

Reply:
It's done.

Reply:
Doesn't seem so.

Reply:
Sorry, I missed the `OmegaDaugthers`. It's done now.

Reply:
Thanks. Why does `kfpBachKaon` start with `kfp` and `kfV0` with `kf`? What is the meaning of `p`?

Reply:
OK, thanks. That looks much better.